### PR TITLE
For the love of god.

### PR DESCRIPTION
**Pull request by malted**
Hold it together for fuck's sake

### CODE REVIEW

**Comment by kekkodance** [comment]
the best client mod ever

**Comment by SlippingGitty** [comment]
2tbwf7

**Comment by HonbraDev** [comment]
fucky wucky better imo

**Comment by Captain8771** [comment]
I disagree with this pull request, why? Well you see I use Arch Linux and its great. it might be very hard at first, but you will get used to it. You see, Arch Linux is a very lightweight operating system. It doesn't come with bloatware pre installed, which is great, but it doesn't come with a desktop environment pre installed which is kind of sad to be honest. But, I can just install it myself, and enjoy using my computer.

Oh, and I agree with what the above person said.

**Comment by Craftzman7** [comment]
I agree with what the above person said.

**Comment by zghwi** [comment]
I AGREE WITH MALTED

**Comment by ShuchirJ** [comment]
fucky wucky way faster to recgonize tho

**Comment by malted** [comment]
I'd just like to interject for a moment. What you're refering to as fucky wucky, is in fact, GNU/fucky wucky, or as I've recently taken to calling it, GNU plus fucky wucky. fucky wucky is not an operating system unto itself, but rather another free component of a fully functioning GNU system made useful by the GNU corelibs, shell utilities and vital system components comprising a full OS as defined by POSIX.

Many computer users run a modified version of the GNU system every day, without realizing it. Through a peculiar turn of events, the version of GNU which is widely used today is often called fucky wucky, and many of its users are not aware that it is basically the GNU system, developed by the GNU Project.

There really is a fucky wucky, and these people are using it, but it is just a part of the system they use. fucky wucky is the kernel: the program in the system that allocates the machine's resources to the other programs that you run. The kernel is an essential part of an operating system, but useless by itself; it can only function in the context of a complete operating system. fucky wucky is normally used in combination with the GNU operating system: the whole system is basically GNU with fucky wucky added, or GNU/fucky wucky. All the so-called fucky wucky distributions are really distributions of GNU/fucky wucky!

**Comment by zghwi** [comment]
merge this pr or i will tell dwayne johnson to not twerk on your birthdays

**Comment by Captain8771** [comment]
do not merge this pr

**Comment by analogical-bytes** [comment]
Made your copypasta better:

I'd just wike to intewject fow a moment. What you'we wefewwing to as fucky wucky, is in fact, gnu/fucky wucky, ow as i've wecentwy taken to cawwing it, gnu pwus fucky wucky. fucky wucky is not an opewating system unto itsewf, but wathew anothew fwee component of a fuwwy functioning gnu system made usefuw by the gnu cowewibs, sheww utiwities and vitaw system components compwising a fuww os as defined by posix.

many computew usews wun a modified vewsion of the gnu system evewy day, without weawizing it. Thwough a pecuwiaw tuwn of events, the vewsion of gnu which is widewy used today is often cawwed "fucky wucky", and many of its usews awe not awawe that it is basicawwy the gnu system, devewoped by the gnu pwoject.

thewe weawwy is a fucky wucky and these peopwe awe using it, but it is just a pawt of the system they use. fucky wucky is the kewnew: the pwogwam in the system that awwocates the machine's wesouwces to the othew pwogwams that you wun. the kewnew is an essentiaw pawt of an opewating system, but usewess by itsewf; it can onwy function in the context of a compwete opewating system. fucky wucky is nowmawwy used in combination with the gnu opewating system: the whowe system is basicawwy gnu with fucky wucky added, ow gnu/fucky wucky . Aww the so-cawwed "fucky wucky" distwibutions awe weawwy distwibutions of gnu/fucky wucky.


Also pls dont merge the pr

**Comment by Captain8771** [comment]
Jesus Christ

**Comment by aetheryx** [comment]
:blobpopcorn:

**Review by Captain8771 — CHANGES_REQUESTED** [review]
changing the message to fucky wucky would be beneficial to the economy, could you implement that?

File: injectors/index.js [hunk 1]
```diff
@@ -113,6 +113,6 @@ const VALID_PLATFORMS = [ 'stable', 'ptb', 'canary', 'dev', 'development' ];
     );
     console.log('Try again with elevated permissions.');
   } else {
-    console.error('fucky wucky', e);
+    console.error('An unknown error occured. Please try again.', e);
```
Comment (Captain8771) [inline injectors/index.js]:
```suggestion
    console.error('fucky wucky', e);
```

**Review by Captain8771 — CHANGES_REQUESTED** [review]
could this be inplemented

File: injectors/index.js (hunk 1 above)
Comment (cnnrde) [reply injectors/index.js]: this implementation would be beneficial to the economy

Comment (Captain8771) [reply injectors/index.js]: it would also delete NFT's

**Comment by MulverineX** [comment]
blame: (aeth)
image

File: injectors/index.js (hunk 1 above)
Comment (smugvee) [inline injectors/index.js]:
```suggestion
    console.error('Uh oh we made a fucky wucky UwU, Pwease try again X3', e);
```

**Comment by zghwi** [comment]
tf

**Review by hayden-droid — APPROVED** [review]

**Comment by zoey-on-github** [comment]
fucky wucky ftw

**Review by hope3x — APPROVED** [review]
I agree.

**Comment by Craftzman7** [comment]
peace at last

**Comment by malted** [comment]
heh

**Comment by Captain8771** [comment]
NOOOOOOOOOO

**Comment by HonbraDev** [comment]
> NOOOOOOOOOO

yes!

**Pull request closed without merging**